### DISCUSSION TITLE
API List  Widget -  Show Tags

### DISCUSSION THREAD

**Discussion by UniperMaster** [Azure/api-management-developer-portal#1606] [General]
When not "groups by tags", can the list widget show all tags in a column associate to an api. 

![image](https://user-images.githubusercontent.com/58111142/150546519-0c9991ff-6cbb-428d-b3b1-c968d866c460.png)

or in the details page (Operation Tags are displayed)

![image](https://user-images.githubusercontent.com/58111142/150546968-2856e36d-cfc9-4e6f-9a2b-063955e50cf9.png)

**Comment by UniperMaster** [comment]
Maybe add it as an options ,  "show tags", this can be disable in the group by view

![image](https://user-images.githubusercontent.com/58111142/150548325-5536eb95-8b85-4ab0-b5be-4a5b06e767b0.png)

**Comment by mikebudzynski** [comment]
@UniperMaster, could you please open an Issue for this request? The "Discussions" part is only for discussions, not bug reports or feature requests.
